src/ixdat/exporters/csv_exporter.py

```diff
@@ -84,6 +85,8 @@ def prepare_header_and_data(self, measurement, v_list, tspan):
             s_list.append(v_name)
 
         header_lines = []
+        ixdat_version_line = f"ixdat version = {__version__}\n"
```

Comment:
You can also choose to version the file format, depending on what is most convenient

Reply:
this is most convenient.